packages/selection/src/types.ts

```diff
@@ -34,6 +34,8 @@ export interface IUseSelectionProps<Value> {
    * @param focusedValue The focused value
    */
   onFocus?: (focusedValue?: Value) => void;
+  /** @ignore */
```

Comment:
I think it's totally fine to expose and document this container prop, but I'm fine to go either way.